src/enzo/Star_ComputePhotonRates.C

```diff
@@ -61,17 +64,45 @@ int Star::ComputePhotonRates(const float TimeUnits, int &nbins, float E[], doubl
     E[2] = 58.0;
     E[3] = 12.8;
     _mass = max(min((float)(_mass), 500), 5);
-    if (_mass > 9 && _mass <= 500) {
-      Q[0] = pow(10.0, 43.61 + 4.9*x   - 0.83*x2);
-      Q[1] = pow(10.0, 42.51 + 5.69*x  - 1.01*x2);
-      Q[2] = pow(10.0, 26.71 + 18.14*x - 3.58*x2);
-      Q[3] = pow(10.0, 44.03 + 4.59*x  - 0.77*x2);
-    } else if (_mass > 5 && _mass <= 9) {
+    if (_mass > 5 && _mass < 9) {
       Q[0] = pow(10.0, 39.29 + 8.55*x);
       Q[1] = pow(10.0, 29.24 + 18.49*x);
       Q[2] = pow(10.0, 26.71 + 18.14*x - 3.58*x2);
       Q[3] = pow(10.0, 44.03 + 4.59*x  - 0.77*x2);
-    } // ENDELSE
+    }
+    else if (_mass >= 9 && _mass <= 120) {
+      if (PopIII_Rotating == 1) {
```

Comment:
For some set of parameters, we should leave in the Schaerer (2002) photon luminosities to give a consistent model with the other mass ranges and to reproduce previous simulations. Here we're replacing them with the new non-rotational models.